.github/workflows/codeql.yml

```
@@ -0,0 +1,42 @@
+      security-events: write
+      packages: read
+      actions: read
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: ['javascript-typescript']
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)